### PR TITLE
Backport changes from gerrit to loki-script

### PR DESCRIPTION
**Pull request by mattclarke**
This contains the changes I had to make to satisfy the various gerrit checks.

Let's back-port the changes so when/if the gerrit version is merged we don't get too many merge conflicts.

### CODE REVIEW

File: nicos_ess/loki/gui/loki_scriptbuilder.py [hunk 1]
```diff
@@ -429,16 +427,12 @@ def _extract_labeled_data(self):
 
     @pyqtSlot()
     def on_generateScriptButton_clicked(self):
-        if self.is_data_in_hidden_columns():
```
Comment (ebadkamil) [inline nicos_ess/loki/gui/loki_scriptbuilder.py:432]: This warning is not needed?

**Comment by ebadkamil** [comment]
LGTM.

**Review by ebadkamil — APPROVED** [review]

File: nicos_ess/loki/gui/loki_scriptbuilder.py (hunk 1 above)
Comment (mattclarke) [reply nicos_ess/loki/gui/loki_scriptbuilder.py:432]: Copy and paste error. Will fix, thanks!

File: nicos_ess/loki/gui/loki_script_generator.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def _start_sample(row_values):
     script += _get_command(row_values.get('pre-command'))
     script += (f'set_sample(\'{row_values["sample"]}\', '
                f'{row_values["thickness"]})\n')
-    script += f'set_position({row_values["position"]})\n'
+    script += f'move(positioner, {row_values["position"]})\n'
```
Comment (ebadkamil) [inline nicos_ess/loki/gui/loki_script_generator.py:69]: One more thing. `positioner` here is not defined.

Comment (mattclarke) [reply nicos_ess/loki/gui/loki_script_generator.py:69]: Yeah, it is just a placeholder like temperature above

Comment (mattclarke) [reply nicos_ess/loki/gui/loki_script_generator.py:69]: Means we cannot do a "dry-run" without creating a device, but that is simple enough

**Pull request merged**